getting-set-up.qmd

```diff
@@ -23,7 +23,7 @@ The course relies on running stan through the `cmdstanr` **R** package, which it
 This requires a separate installation step:
 
 ```{r cmdstan_install, eval = FALSE}
-cmdstanr::install_cmdstan()
+cmdstanr::install_cmdstan(cores = 2)
```

Comment:
```suggestion
cmdstanr::install_cmdstan()
```

Reply:
I don't really understand why we would want people to have to wait longer when installing but I also don't really mind